boards/silabs/radio_boards/xg29_rb4412a/Kconfig.defconfig

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2025 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_XG29_RB4412A
+
+config LOG_BACKEND_SWO_FREQ_HZ
+	default 875000
+	depends on LOG_BACKEND_SWO
+
+if SOC_GECKO_USE_RAIL
+
+config FPU
+	default y
+
+endif # SOC_GECKO_USE_RAIL
```

Comment:
If the condition is trivial, it is not useful to repeat it after `endif`

Reply:
I disagree that this one is trivial -- it's the end of the inner block in a nested pair of blocks. Since the Kconfig file format doesn't use indentation, marking which block ends here with a comment is useful information IMO.